src/java/com/unifina/signalpath/AbstractJavaCodeWrapperModuleException.java

```diff
@@ -0,0 +1,26 @@
+package com.unifina.signalpath;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AbstractJavaCodeWrapperModuleException extends ModuleException {
+	public AbstractJavaCodeWrapperModuleException(String message, Throwable cause, List<ModuleExceptionMessage> moduleExceptions) {
+		super(message, cause, moduleExceptions);
+	}
+
+	/**
+	 * For front-end JSON output.
+	 */
+	@Override
+	public Map<String, Object> toMap() {
+		List<Map<String, Object>> list = new ArrayList<>();
```

Comment:
This is actually what should be the default implementation in `ModuleException#toMap()`. This class is not needed then?

Reply:
not needed